src/Surfnet/StepupMiddleware/ManagementBundle/Validator/IdentityProviderConfigurationValidator.php

```diff
@@ -27,22 +27,28 @@ public function validate(array $configuration, $propertyPath)
     {
         Assertion::isArray($configuration, 'invalid configuration format, must be an object', $propertyPath);
 
-        $acceptedProperties = [
+        $requiredProperties = [
             'entity_id',
             'loa',
```

Comment:
We need the `use_pdp` configuration parameter in the list of `requiredProperties`. This because the `StepupAssert::keysMatch` throws an exception when the `use_pdp` configuration is handled.

I suggest to add `use_pdp` to the list of `requiredProperties`. 

This also goes for the `IdPConfigurationValidator`

Reply:
I have changed this for now, as we want to release this to the testing environment. Feel free to add to the discussion @jorissteyn or change this to a solution that is more to your liking.

Reply:
I see how we broke this now, merging this one.